platform/core/src/classes/metadata/StudyMetadata.js

```diff
@@ -318,6 +345,9 @@ export class StudyMetadata extends Metadata {
    */
   setDisplaySets(displaySets) {
     if (Array.isArray(displaySets) && displaySets.length > 0) {
+      // TODO: This is weird, can we just switch it to writable: true?
+      this._displaySets.splice(0);
```

Comment:
I think this is part of a "data cleanup" conversation with James. We need to discuss how/when/where we should be able to access and mutate data. If we're doing this anyway, I don't think there's any harm in `writable: true`. I'm not sure what assumptions we would be breaking, unless this impacts reactivity (For example, in vuejs, you have to use `splice` for arrays or the reactivity isn't triggered)

Reply:
I left it as writable: false for now. We can come back to this later. I think it's just to prevent developers from mutating that stuff directly. We want the class to handle it internally.